docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: "3.7"
+services:
+  curity:
+    container_name: idsvr
+    environment:
+      PASSWORD: Secr3t
```

Comment:
Maybe put <ADMIN_PASSWORD> as value so they have to set it to make docker-compose to work?